### DISCUSSION TITLE
TTL API support

### DISCUSSION THREAD

**Discussion by ghost** [hse-project/hse#11] [Ideas]
Hello, 
Is there any api that supports key-value ttl(time to live) in hse?  or full apis docmentation?

**Comment by smoyerx** [comment]
The HSE APIs are fully documented on the [HSE Wiki](https://github.com/hse-project/hse/wiki).  We recommend reviewing the **Concepts** section first, and then the **API Reference**, which per the Wiki link is actually embedded in the **hse.h** header file.

We do not currently support TTL.  This feature is on our roadmap, but I cannot confirm when it will be available.

**Comment by ghost** [comment]
Ok, thx.
